### PR TITLE
[FIX] Unread bottom, top bars on sidebar (#16853)

### PR DESCRIPTION
**Pull request by juzser**
Issue number #16853 

Screen Shot 2020-03-12 at 23 09 13

- Change class `li.has-alert` that never be used, to `li.sidebar-item--unread` to detect the position of unread message room.
- Trigger the `updateUnreadBars` function when receive new unread message.
- Update theme style for the `unread-rooms` that is display: none as default.

### CODE REVIEW

**Comment by sampaiodiego** [comment]
I've been missing these bars for sooooo long now.. @brunosquadros is this something we're considering to have back?

**Comment by JuzSer** [comment]
Yes, no problem, I'm a fan of you guys.
When I tried to make these bars via custom script, I figured out that you guys had done this. 
If there are something wrong, please let me know then I can update the PR. 

I see there is a small problem when you resize the window by height (from have scroll on sidebar to scroll disappeared) so the bar is still there. 
I'm considering to trigger add hidden class to the bar on resize but I feel it's not really neccessary.

File: app/theme/client/imports/components/sidebar/sidebar.css [hunk 1]
```diff
@@ -64,7 +64,9 @@
 	}
 
 	& .unread-rooms {
-		display: none;
+		padding: calc(var(--sidebar-small-default-padding) - 8px);
```
Comment (gabriellsh) [inline app/theme/client/imports/components/sidebar/sidebar.css]: What are those 8 pixels? scrollbar?

Comment (JuzSer) [reply app/theme/client/imports/components/sidebar/sidebar.css]: No, the `.sidebar-item` has 3px padding top and the total height is 34px (base on `var(--sidebar-item-height-medium)`). 
So I put the default padding there plus 8px to equal the `sidebar-item`

**Comment by JuzSer** [comment]
So do I need to update anything?

**Comment by ggazzo** [comment]
hey, @JuzSer thanks for your time, I don't we any changes, I will check with design and  we gonna merge your pr :) soon as possible


ps: pay attention, you opened a pr to master, you should work always on the develop branch ... thanks again...

**Comment by JuzSer** [comment]
Nice. I see you pick my commit to `develop`, thank you. 👍🏼
Next time I will work on `develop` branch.

**Pull request merged**